defaults/main.yml

```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
   account_first: true
   strip_v1: true
   swift3_compat: true
-  create_dir_placeholders: true
-  recursive_placeholders: true
+#  create_dir_placeholders: true
+#  recursive_placeholders: true
+  sentinel_hosts:
+    "{{ groups['openio_redis_cluster'] \
```

Comment:
Replace this jinja by a default value like `192.168.1.1:6012,192.168.1.2:6012,192.168.1.3:6012`
Playbook will map extract the needed value.